serverless/pages/manage-org.asciidoc

```diff
@@ -1,15 +1,211 @@
 [[general-manage-organization]]
 = Manage your organization
 
-// :description: Manage your instances, users, and settings.
+// :description: Manage your Elastic Cloud organization.
 // :keywords: serverless, general, organization, overview
 
 preview:[]
 
-When you sign up to Elastic Cloud, you create an **organization**.
+An **organization** is the umbrella for all of your {ecloud} resources, including hosted deployments and serverless projects, users, and account settings. When you sign up to {ecloud}, you create an organization.
 
-This organization is the umbrella for all of your Elastic Cloud resources, users, and account settings. Every organization has a unique identifier. Bills are invoiced according to the billing contact and details that you set for your organization.
+In this article we walk you through the essential processes to effectively oversee your organization.
 
-* <<general-manage-access-to-organization>>: Add members to your organization and projects.
-* <<general-manage-billing>>: Configure the billing details of your organization.
-* <<elasticsearch-manage-project>>: Configure project-wide features and usage.
+Learn how to:
+
+* <<general-user-profile>>: Manage or update your user profile.
+* <<general-manage-access-to-organization>>: Invite users in your organization to access serverless projects and specify their roles.
+* <<general-assign-user-roles>>: Assign predefined roles to users in your organization.
+* <<general-join-organization-from-existing-cloud-account>>: Join a new organization and bring over your projects.
+* <<general-leave-an-organization>>: Leave an organization.
+
+[discrete]
+[[general-user-profile]]
+== Update your user profile
```

Comment:
I think this isn't related to managing the org, it's the actual only "personal" settings view in the product. And it's not serverless specific. I'd move it to its own page, or simply refer to the cloud version of it which would essentially be the same (and actually maintained)